modules/administration-guide/partials/con_gateway.adoc

```diff
@@ -1,7 +1,12 @@
 [id="gateway_{context}"]
 = Gateway
 
-The {prod-short} gateway is a Traefik instance applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.
+The {prod-short} gateway consists of 3 main parts:
+
+* *link:https://github.com/traefik/traefik[Traefik]* instance responsible for requests routing
+* *link:https://github.com/oauth2-proxy/oauth2-proxy[OAuth2-proxy]* ensuring authentication and initiating OAuth flow
+* *link:https://github.com/brancz/kube-rbac-proxy[kube-rbac-proxy]* applying {orch-name} Role based access control (RBAC) policies to control access to any {prod-short} resource.
+
```

Comment:
```suggestion

{link-oidc} is the default OIDC issuer. The {prod-short} operator deploys it with preconfigured static users. It can serve as a bridge to third party {identity-provider}, such as GitHub.

```

and define the link in `antora.yml`:

```
    link-oidc-issuer: link:https://dexidp.io/[Dex]
```